lib/ACL/ACLStorageWrapper.php

```diff
@@ -176,10 +175,8 @@ public function writeStream(string $path, $stream, ?int $size = null): int {
 
 	/**
 	 * @inheritDoc
-	 * @param string $path
-	 * @param ?IStorage $storage
 	 */
-	public function getCache($path = '', $storage = null): ICache {
+	public function getCache(string $path = '', ?\OCP\Files\Storage\IStorage $storage = null): ICache {
```

Comment:
We should apply the auto-import as in server first, to avoid it removing the `use`.